pyanaconda/modules/subscription/subscription_data.py

```diff
@@ -173,49 +176,51 @@ def as_dict(self):
             "consumed_entitlement_count" : self.consumed_entitlement_count
         }
 
+    def _pretty_date(self, date_from_json):
+        """Return pretty human readable date based on date from the input JSON."""
+        # fallback in case of the parsing fails
+        date_string = date_from_json
+        try:
+            # The start/end date in GetPools() output seems to be formatted as
+            # "Locale’s appropriate date representation.".
+            date = datetime.datetime.strptime(date_from_json, "%m/%d/%y")
+            # get a nice human readable date
+            date_string = date.strftime("%b %d, %Y")
+        except ValueError:
+            log.debug("subscription date parsing failed: %s", date_from_json)
```

Comment:
I would guess this is a warning.

Reply:
Good point, warning makes more sense here.